ksqldb-engine/src/main/java/io/confluent/ksql/engine/rewrite/QueryAnonymizer.java

```diff
@@ -399,18 +392,6 @@ public String visitTerminateQuery(final TerminateQueryContext context) {
       return "TERMINATE query";
     }
 
-
-    @Override
-    public String visitDescribeStreams(final DescribeStreamsContext context) {
```

Comment:
Why do you remove this one? Should be sufficient to only drop the 3 line of the `if "extended"` piece.